src/OpenTelemetry.Contrib.Instrumentation.AWS/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Raised the minimum .NET version to `net462`
```

Comment:
Nit: alternate suggestion:
"Removes .NET Framework 4.5.2, .NET 4.6 support. The minimum .NET Framework version supported is .NET 4.6.2. "

Reply:
I copied the phrasing from
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/b06a6f290fc59a292994e2ba9973eefe0691fa19/src/OpenTelemetry.Contrib.Extensions.AWSXRay/CHANGELOG.md?plain=1#L8-L9

Should I still change it to your suggested phrasing?

Reply:
No! I copied from https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md#130-beta1 and felt it slightly better. But it is very subjective, and it should not block this PR!